skyxplore-ui-test/src/test/java/com/mentoring/pageobject/Page.java

```diff
@@ -4,12 +4,7 @@
 import com.mentoring.framework.utils.AllureLogger;
 import io.qameta.allure.Step;
 import lombok.extern.slf4j.Slf4j;
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
-import org.openqa.selenium.NoSuchElementException;
-import org.openqa.selenium.TimeoutException;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
```

Comment:
Please do not use such import. Change your setting in IntelliJ:

![image](https://user-images.githubusercontent.com/13760531/85985162-a205db00-b9ea-11ea-93d8-028501660460.png)

Reply:
ok, modified, thx